src/software/thunderscope/requirements.txt

```diff
@@ -1 +1,2 @@
 pyqtgraph==0.13.3
+numpy==1.24.4
```

Comment:
This is already in `ubuntu20_requirements.txt` and `ubuntu22_requirements.txt`, why do we need it here?

Reply:
Bazel sandboxes each build I think, so when you set up thunderscope_main, it won't have numpy until you set it as a requirement

Reply:
I believe that was added by @AdenC123.

Reply:
As arun said- I was unable to run thunderscope_main directly without fixing the numpy version here